### DISCUSSION TITLE
How to use `tar_visnetwork` when `_targets.R` is changed to another R script ?

### DISCUSSION THREAD

**Discussion by JauntyJJS** [ropensci/targets#1010] [General]
My humble apologies if I am asking a trivial question.

I tried to create multiple projects using `targets` following the instructions in https://books.ropensci.org/targets/projects.html . From my understanding the default `_targets.R` file is replaced with `script_a.R`

![image](https://user-images.githubusercontent.com/9066508/211994841-995121e5-bc2b-4f61-8c25-2ac4d6a5ec7e.png)

Following the instructions, I managed to configure the project and run `tar_make` using the below codes (using the alternative method)

![image](https://user-images.githubusercontent.com/9066508/211995100-884c4f98-6d6b-4dc4-9017-d39b43c99475.png)

However, I am puzzled on how to use `tar_visnetwork` to view the pipelines for `script_a.R` and `script_b.R`.

I understand this will not work with the following expected error.

```r
tar_visnetwork()

! could not find file _targets.R. Main functions like tar_make() require a target script file (default: _targets.R) to define the pipeline. Functions tar_edit() and tar_script() can help.
```

How do I tell `tar_visnetwork` to look at `script_a.R ? Is it possible to do this without setting any environmental variables ?

Thank you.

**Comment by wlandau** [comment]
`tar_visnetwork()` understands the same `script` and `store` arguments as `tar_make()`, so you can run `tar_visnetwork(script = "scirpt_a.R", store = "store_a")`.

**Reply by JauntyJJS** [reply]
Noted with thanks.
